examples/server/tests/features/steps/steps.py

```diff
@@ -883,7 +883,7 @@ async def request_completion(prompt,
                                     "cache_prompt": cache_prompt,
                                     "id_slot": id_slot,
                                     "seed": seed if seed is not None else 42,
-                                    "temperature": temperature if temperature is not None else "0.8f",
+                                    "temperature": temperature if temperature is not None else 0.8f,
```

Comment:
```suggestion
                                    "temperature": str(temperature) if temperature is not None else 0.8f,
```

In order to always pass the same data type.